public/src/app/app.component.ts

```diff
@@ -269,12 +269,14 @@ export class AppComponent {
     };
 
     let sortMilestones = (a,b) => {
-      const sprintMilestone = /^[A-Z](\d+)S(\d+)$/;
+      const sprintMilestone = /^([A-Z])(\d+)S(\d+)$/;
       let a0 = sprintMilestone.exec(a.title), b0 = sprintMilestone.exec(b.title);
       if(a0 !== null && b0 === null) return -1;
       if(a0 === null && b0 !== null) return 1;
       if(a0 === null) return a.title.localeCompare(b.title);
-      return (parseInt(a0[1], 10) - parseInt(b0[1], 10))*100 + (parseInt(a0[2], 10) - parseInt(b0[2], 10));
+      return (parseInt(a0[2], 10) - parseInt(b0[2], 10))*1000 +
+             (a0[1].charCodeAt(0)-b0[1].charCodeAt(0))*100 +
+             (parseInt(a0[3], 10) - parseInt(b0[3], 10));
```

Comment:
I think this will still produce the wrong order for: `15.0` `B14S8`

Reply:
`15.0` is supposed to go at the end -- it's not a sprint milestone.

Reply:
yes, but does it do it? Currently it's sorted at the front, and I don't see how this change would change this.

![Screenshot from 2021-03-08 12-21-21](https://user-images.githubusercontent.com/181336/110314949-d36a8800-8008-11eb-89d4-bfb04f83db51.png)

Reply:
![image](https://user-images.githubusercontent.com/4498365/110368717-8b497500-809d-11eb-9600-f588e14f0ba6.png)

Unrelated Note: we shouldn't have anything under `15.0` milestone yet as we have not yet triaged for 15.0 -- only the `A15S3` groundwork issues that we discussed. 😉  The `15.0` issues should be under `Future`.

Reply:
:+1: 

> Unrelated Note: we shouldn't have anything under 15.0 milestone yet as we have not yet triaged for 15.0 -- only the A15S3 groundwork issues that we discussed. wink The 15.0 issues should be under Future.

So how do we mark those that we think should be in 15?

Reply:
For now, just put a note in the 15.0 planning document in the Suggestions section.